### DISCUSSION TITLE
How could someone use rodi in web frameworks other than BlackSheep

### DISCUSSION THREAD

**Discussion by aeb-dev** [Neoteroi/rodi#67] [Q&A]
I am trying to integrate rodi with FastAPI but I am not sure how should I approach to the problem.

Is there any example? Flask or any other web framework example would also help

**Comment by lucas-labs** [comment]
@aeb-dev 
I'm creating a library using Rodi and FastAPI. Although it might be overkill for what you're asking, you can see how I implemented and combined them here if you have the time to go through the code haha:

https://github.com/lucas-labs/pest/blob/master/pest/core/module.py

**Comment by aeb-dev** [comment]
Thank you @lucas-labs. We actually inspired from your project to achieve what we want. Thanks for the idea!

**Comment by StummeJ** [comment]
We integrated Rodi into FastAPI via a custom middleware, and to Strawberry via a SchemaExtension.  To not interfere with either framework we use wrappers to remove the autowired parameters from what the frameworks see.  Strawberry specifically isn't friendly to this process.
